modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FDSCustomView.macro

```diff
@@ -10,8 +20,8 @@ definition {
 		Click(locator1 = "FrontendDataSet#SELECT_CUSTOM_VIEW");
 
 		Click(
-			key_optionName = "${key_itemName}",
-			locator1 = "FrontendDataSet#SELECT_OPTION_CUSTOM_VIEW");
```

Comment:
keep `key_optionName`

our standard naming is `key_${descriptor}` when we need to pass a variable to the xpath.